tasks/tests/integration/test_timeseries_backfill.py

```diff
@@ -43,36 +51,46 @@ async def test_backfill_dataset_run_async(dbsession, mocker, mock_storage):
         mock_storage.write_file("archive", master_chunks_url, content)
 
     task = TimeseriesBackfillCommitsTask()
+    dataset_names = [
+        MeasurementName.coverage.value,
+        MeasurementName.flag_coverage.value,
+    ]
     res = await task.run_async(
         dbsession,
         commit_ids=[commit.id_],
-        dataset_names=[
-            MeasurementName.coverage.value,
-            MeasurementName.flag_coverage.value,
-        ],
+        dataset_names=dataset_names,
     )
     assert res == {"successful": True}
-
-    coverage_measurement = (
-        dbsession.query(Measurement)
-        .filter_by(
-            name=MeasurementName.coverage.value,
-            commit_sha=commit.commitid,
-        )
-        .one_or_none()
+    mocked_app.tasks[
```

Comment:
I'm unsure if this makes sense here since this is an integration test. We used to call the function and wait for it here, https://github.com/codecov/worker/pull/256/files#diff-902a7b3fa8f6ee724af5423139f45596d96edeb267f72ab8aa8cd47a441c467dR38, but now we're dispatching tasks to do the logic, so the previous test logic I removed didn't await for the changes to take place. Would it make sense here to await for each task to finish then? (don't think so), or is this test adjustment good?